src/ui/Api.js

```diff
@@ -11,12 +11,12 @@ export default class Api {
       // here we assume we want to create a new deck
       const resp = await fetch("/api/v2/deck/new", { method: "POST" });
       const response = await resp.json();
-      return response;
+      return "response";
```

Comment:
Why are we quoting these? Is this just temporary? I suppose you know that it's just returning the string `"response"` now instead of a JSON object. :)